src/components/tasks/card/index.tsx

```diff
@@ -85,7 +85,7 @@ const Card: FC<Props> = ({
   
   function handleProgressColor(percentCompleted: number, startedOn: string, endsOn: string): string {
     const percentageofDays = getPercentageOfDays(startedOn, endsOn)
-    if(percentageofDays >= percentCompleted) {
+    if(percentCompleted === 100 && (percentageofDays >= percentCompleted || percentageofDays < 0)) {
```

Comment:
This condition looks to be incorrect. 

If I have a task for a total of 10 days, and it's the 5th day today, but I have completed 80% of the task, then the progress bar should be green.

Reply:
So in that case the percentage of days is greater than the percentage completed so it will show green

Reply:
` percentCompleted === 100 && (percentageofDays >= percentCompleted || percentageofDays < 0)` will become falsy for `percentCompleted=70, but percentageofDays=50.

The task is going well, because it's 70% done and we still have more than half the time left. It should be green.

Reply:
done